src/why3_tactic.ml

```diff
@@ -180,7 +180,9 @@ let handle : Pos.popt -> Proof.proof_state -> sig_state -> string option ->
   (* Get the meta type of the current goal (with quantified context). *)
   let trm = !((Proof.Goal.get_meta g).meta_type) in
   (* Add the axiom to the current signature. *)
-  let a = Sign.add_symbol ss.signature Const (Pos.none axiom_name) trm [] in
+  let a = Sign.add_symbol ss.signature Public Const (Pos.none axiom_name) trm
+      []
```

Comment:
not very nice ;-) could you try to get it on 1 line? (rename trm into t, or add let p = Pos.none ...)